src/main/java/classify/commands/EditStudent.java

```diff
@@ -0,0 +1,222 @@
+package classify.commands;
```

Comment:
Good use of separation of concerns in creating a new class.
![image](https://github.com/AY2324S2-CS2113-T13-3/tp/assets/110646458/7d3a5ed2-d386-437a-9dd4-9f9ee7fc279a)